guides/common/header.adoc

```diff
@@ -10,6 +10,13 @@ endif::[]
 ifeval::["{DocState}" == "nightly"]
 :revnumber: Nightly
 :revdate: published {date_my}
+++++
+<!-- "Fork me on GitHub" CSS ribbon v0.2.3 | MIT License -->
+<!-- https://github.com/simonwhitaker/github-fork-ribbon-css -->
+<a class="github-fork-ribbon right-top fixed" href="https://docs.theforeman.org" data-ribbon="Pre-release version" title="Pre-release version">
```

Comment:
What's the reason to link to the top level from here? Thinking out loud: is there a logical link or should it not have one? Does this work as well?
```suggestion
<a class="github-fork-ribbon right-top fixed" data-ribbon="Pre-release version" title="Pre-release version">
```

Reply:
Looking at https://github.com/theforeman/foreman-documentation/issues/695#issuecomment-2586593017, you are right: No reason to add the link to the pre-release ribbon.